projects/swarmd/swarmd/Makefile

```diff
@@ -1,15 +1,4 @@
-.PHONY: tag push
-all: push
+ORG?=linuxkitprojects
+include ../../../pkg/package.mk
```

Comment:
`pkg/package.mk` has:
```
ifndef $(NETWORK)
NET_OPT=
else
NET_OPT=--network=none
endif
```
So setting `NETWORK=1` would appear to disable the network. Which is a bit confusing because e.g. `pkg/runc` sets `NETWORK=1` and does a clone, and appears to work fine.

Ah, I see the error to work as the author (@dave-tucker I think?) expected it would need to be `ifndef NETWORK`, as it is it is expanding to either `ifndef` (i.e. no argument at all) or `ifndef 1` (if `NETWORK=1`) both of which apparently evaluate to true (if it were possible ot assign to a variable named `1` things might be different).

Rather than fix the syntax here thought I'd suggest we fix the semantics and use `ifneq ($(NETWORK),1)` (or `y`) which I think will match what people expect. I'll add that to #2117 I guess, I'll also add the NETWORK=1 here in preparation.